showcase/gapic-showcase/src/main/java/com/google/showcase/v1beta1/stub/HttpJsonEchoStub.java

```diff
@@ -366,7 +368,24 @@ protected HttpJsonEchoStub(
       throws IOException {
     this.callableFactory = callableFactory;
     this.httpJsonOperationsStub =
-        HttpJsonOperationsStub.create(clientContext, callableFactory, typeRegistry);
+        HttpJsonOperationsStub.create(
```

Comment:
I guess the actual showcase integration tests that verified the interactions will be done in this [PR](https://github.com/googleapis/gapic-generator-java/pull/1431/files#diff-22c84236080a30cc7ccf014fadf1a077821e97f1a4b46f1543e25ccc1075425b)?

Reply:
Going forward, the interactions will be covered in that PR. For this PR, I just generated the speech client locally and tested that Operations works.